source/install/docker-local-machine.rst

```diff
@@ -95,27 +95,10 @@ Fedora
        
 3. When Docker is done fetching the image, open http://localhost:8065/ in your browser.
 
-Arch
-^^^^
-
-1. Install Docker using the following commands:
-
-   .. code:: bash
-
-       pacman -S docker
-       systemctl enable docker.service
-       systemctl start docker.service
-       gpasswd -a <username> docker
-       newgrp docker
-
-2. Start Docker container:
-
-   .. code:: bash
-
-       docker run --name mattermost-preview -d --publish 8065:8065 mattermost/mattermost-preview
+Arch Linux
+^^^^^^^^^^
 
-3. When Docker is done fetching the image, open ``http://localhost:8065/``
-   in your browser.
+`Use the following online guide from the Arch Linux wiki<https://wiki.archlinux.org/index.php/Mattermost>`_.
```

Comment:
Ok it seems you made suggestions directly on the files using the github comments system. Waiting for your reply to before making any changes if any.